components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -211,7 +211,34 @@ func (backend *ESClient) DeleteProjectTag(ctx context.Context,
 }
 
 func (backend *ESClient) DeleteReportProjectTag(ctx context.Context, projectTagToDelete string) (string, error) {
-	return "", nil
+	script := `
+		if (ctx._source['projects'] != null) {
+			def foundIndex = ctx._source.projects.indexOf(params.project);
+
+			if (foundIndex != -1) {
+				ctx._source.projects.remove(foundIndex);
+			}
+		}
+	`
+
+	params := map[string]interface{}{"project": projectTagToDelete}
+	docType := mappings.DocType
+	mapping := mappings.ComplianceRepDate
+	index := fmt.Sprintf("%s-%s", mapping.Index, "*")
+
+	startTaskResult, err := elastic.NewUpdateByQueryService(backend.client).
+		Index(index).
+		Type(docType).
+		Script(elastic.NewScript(script).Params(params)).
```

Comment:
@lancewf under the UpdateReportProjectTag, the query includes `Refresh("true").` Do you know if we need to refresh the indices after deleting a project tag?

Reply:
I would say we would want to refresh to ensure the delete is live when we say this task is complete. Since deletes will not happen that often a refresh should not add too much load to Elasticsearch.